src/einsteinpy/symbolic/predefined/vacuum_solutions.py

```diff
@@ -66,6 +66,43 @@ def Kerr(c=constants.c, sch=symbols("r_s"), a=symbols("a")):
 
 
 def KerrNewman(
-    c=constants.c, G=constants.G, sch=symbols("r_s"), a=symbols("a"), Q=symbols("Q")
+    c=constants.c, G=constants.G, eps_0=constants.eps_0, sch=symbols("r_s"), a=symbols("a"), Q=symbols("Q")
 ):
-    raise NotImplementedError
+    """
+    Kerr-Newman Metric in Boyer Lindquist coordinates.
+
+    Parameters
+    ----------
+    c : ~sympy.core.basic.Basic or int or float
+        Any value to assign to speed of light. Defaults to ``c``.
+    eps0 : ~sympy.core.basic.Basic or int or float
+        Any value to assign to the electric constant or permittivity of free space. Defaults to ``eps0``.
+    sch : ~sympy.core.basic.Basic or int or float
+        Any value to assign to Schwarzschild Radius of the central object. 
+        Defaults to ``r_s``.
+    a : ~sympy.core.basic.Basic or int or float
+        Spin factor of the heavy body. Usually, given by ``J/(Mc)``,
+        where ``J`` is the angular momentum.
+        Defaults to ``a``.
+    Q:  ~sympy.core.basic.Basic or int or float
+
+    """
+    coords = symbols("t r theta phi")
+    t, r, theta, phi = coords
+    Sigma = r ** 2 + (a ** 2 * cos(theta) ** 2)
+    rQsq = ((Q**2)* G) / (4 * pi * eps_0 * (c**4))
+    Delta = r ** 2 - sch * r + a ** 2+rQsq
+    c2 = c ** 2
+
+    metric = diag(
+        1 - (sch * r / Sigma),
```

Comment:
This `metric` defination seems same to that of Kerr aside from a different  `Delta`. Are you sure two are the same?

I know that Kerr is a specific case of KerrNewman, but does both of them contains the same expression?. I don't know because I do not have extensive background in GR.

Reply:
Indeed in Boyer-Linquist coordinate this is the case. Maybe another review could confirm this, since we want to double check things.